spec/requests/users_spec.rb

```diff
@@ -1,17 +1,30 @@
 require 'rails_helper'
 
-RSpec.describe 'Users', type: :request do
-  describe 'GET /index' do
-    it 'returns http success' do
-      get '/users/index'
-      expect(response).to have_http_status(:success)
+RSpec.describe UsersController, type: :controller do
+  describe 'GET #index' do
```

Comment:
- [ ] [REQUIRED] Good job so far! Spec file include tests that ensure the response body includes the correct placeholder text for `user_spec`. Example : `expect(response.body).to include("Welcome to user index page")`, please implement it 🙏🏼